SU2_CFD/include/variables/CVariable.hpp

```diff
@@ -1951,6 +1951,10 @@ class CVariable {
    */
   inline virtual CVectorOfMatrix& GetGradient_Reconstruction(void) { return Gradient; }
 
+  //virtual inline void uselesstestfunction(void) {  }
+
+
+
```

Comment:
I think the name says what needs to be done here :)

Reply:
oops. :)